package.json

```diff
@@ -31,6 +32,7 @@
 		"protobuf",
 		"protocol buffers",
 		"buf",
+		"bufbuild",
 		"lint"
```

Comment:
If `lint` is here, let's add `format`, too.

Reply:
done